doc/dialects/exasol.md

```diff
@@ -0,0 +1,69 @@
+# Exasol SQL Dialect
+
+Connecting to an Exasol database is the simplest way to get started with Virtual Schemas.
+You don't have to install any JDBC driver, because it is already installed in the Exasol database and also included in the JAR of the JDBC adapter.
```

Comment:
This reminds me that we can now probably remove the Exasol JDBC driver from the other dialects.